cmd/launcher/launcher.go

```diff
@@ -386,6 +387,7 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 	osqueryRunner := osqueryruntime.New(
 		k,
 		client,
+		osquerypublisher.NewOsqueryPublisher(k),
```

Comment:
osquery has multiple log types. It's not wrong for us to send them all through the same publisher, but I wanted to point out the assumption.